From f64c06a7b615d3c48cdadbc9679a191e8329a933 Mon Sep 17 00:00:00 2001
From: Dominic Hargreaves <dom@earth.li>
Date: Sun, 13 May 2012 19:10:28 +0100
Subject: Fix Module::CoreList to contain references to the string value
 '5.016000'

This is needed for debian/t/control.t and is expected to be fixed in
the next 5.16 release candidate

Bug: http://www.nntp.perl.org/group/perl.perl5.porters/2012/05/msg186545.html
Patch-Name: fixes/module_corelist_5.16_string.diff
---
 dist/Module-CoreList/lib/Module/CoreList.pm |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/dist/Module-CoreList/lib/Module/CoreList.pm b/dist/Module-CoreList/lib/Module/CoreList.pm
index 9b5b07a..895a9b7 100644
--- a/dist/Module-CoreList/lib/Module/CoreList.pm
+++ b/dist/Module-CoreList/lib/Module/CoreList.pm
@@ -34687,6 +34687,7 @@ $released{'5.011000'} = $released{5.011};
 $released{'5.012000'} = $released{5.012};
 $released{'5.013000'} = $released{5.013};
 $released{'5.014000'} = $released{5.014};
+$released{'5.016000'} = $released{5.016};
 
 $version{'5.000'} = $version{5};
 $version{'5.010000'} = $version{5.01};
@@ -34694,11 +34695,13 @@ $version{'5.011000'} = $version{5.011};
 $version{'5.012000'} = $version{5.012};
 $version{'5.013000'} = $version{5.013};
 $version{'5.014000'} = $version{5.014};
+$version{'5.016000'} = $version{5.016};
 
 $deprecated{'5.011000'} = $deprecated{5.011};
 $deprecated{'5.012000'} = $deprecated{5.012};
 $deprecated{'5.013000'} = $deprecated{5.013};
 $deprecated{'5.014000'} = $deprecated{5.014};
+$deprecated{'5.016000'} = $deprecated{5.016};
 
 1;
 __END__
